algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/models.py

```diff
@@ -81,7 +84,10 @@ def __init__(self, encoder_dim: int = 0, input_dropout_rate: float = 0.0):
     self.conv2 = Conv2dSubsampling(
         input_channels=encoder_dim, output_channels=encoder_dim)
 
-    self.linear = nn.LazyLinear(out_features=self.encoder_dim, bias=True)
+    self.linear = nn.Linear(
+        in_features=self.encoder_dim * num_bins // 4,
```

Comment:
what's the reasoning for this?

Reply:
Each of the two subsampling layers reduce the number mel-spectrogram features by half.

Reply:
but then how is this numerically equivalent to the old model?